CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- moved the `around` logic to more specific interfaces
+- Removed `docs-ui.concepts` from `interfaces.json`
+- Renamed `docs-wrapper` to `navigation-wrapper`
```

Comment:
```suggestion
### Changed
- Moved the `around` logic to more specific interfaces.
- Renamed `docs-wrapper` to `navigation-wrapper`.

### Removed
- `docs-ui.concepts` from `interfaces.json`.
```

Reply:
That's great! Always learning new things about CHANGELOG with your reviews. Thanks!